src/test/java/xbot/common/trajectory/LowResFieldTest.java

```diff
@@ -57,6 +59,7 @@ private FieldPose flipFieldPose(FieldPose current) {
                 new WrappedRotation2d(180 - current.getHeading().getDegrees()));
     }
 
+    @Ignore
```

Comment:
⭐ could you add a comment about why these are ignored? Is it a TODO to get them passing again?
eg:
`@Ignore("Broken from change X, TODO to get passing again.")`